modules/Hoot/managers/api.js

```diff
@@ -349,6 +351,25 @@ export default class API {
             .then( resp => resp.data );
     }
 
+    cancelUpload( cancelToken ) {
+
+        let cancelURL = `${this.baseUrl}/ingest/ingest/upload`;
+
+        axios.get( cancelURL, { cancelToken: cancelToken.token } )
+        .catch(thrown => {
+            if (axios.isCancel(thrown)) {
+            console.log(thrown.message);
+            } else {
+            let alert = {
+                message: 'Failed to cancel upload.',
+                type: 'warn'
+            };
+            Hoot.message.alert( alert );
+            }
+        });
+        cancelToken.cancel('Request cancelled.');
+    }
+
```

Comment:
We don't need a separate method in api.js and don't need to make another axios request.
Instead, just call 
`cancelToken.cancel('Request cancelled.');`
in import.js line 527.